src/TSTransformer/util/types.ts

```diff
@@ -198,18 +198,6 @@ export function getFirstConstructSymbol(state: TransformState, expression: ts.Ex
 	}
 }
 
-export function getFirstDefinedSymbol(state: TransformState, type: ts.Type) {
```

Comment:
Why not just rename and redefine this method? It looks like otherwise you're just doing a lot of long, duplicate function calls that w9ould be easier to read with `getNonOptionalTypeSymbol`